osmaxx/conversion/constants/output_format.py

```diff
@@ -1,7 +1,7 @@
 import uuid
 from collections import OrderedDict, namedtuple
 
-from django.utils.translation import gettext as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
Because with `gettext`, 34aa0e0 would lead to

    INTERNALERROR> django.core.exceptions.AppRegistryNotReady: The translation infrastructure cannot be initialized before the apps registry is ready. Check that you don't make non-lazy gettext calls at import time.

due to `output_format` [running `_(...)` at import time](https://github.com/geometalab/osmaxx/pull/864/files#diff-fc533de93246bf0b7f52b65e33f0ede5R45) and being imported all over the place.

(I'm not quite sure why e.g. the [import in `models.py`](https://github.com/geometalab/osmaxx/pull/864/files#diff-26f3feed2376939bed572ca895031e8fL11) didn't trigger this error even before.)